src/components/DNSenumTool/DNSenumTool.tsx

```diff
@@ -109,6 +111,24 @@ const DnsenumTool = () => {
      * @param {FormValuesType} values - The form values, containing the domain, threads, pages, scrap, reverse lookup, and timeout.
      */
 
+    const checkAllCommandsAvailability = async () => {
```

Comment:
Interesting way of implementing this. I am getting the following: 

![image](https://github.com/user-attachments/assets/a4e40ae9-4611-4a83-9fa4-1d5a1217f1a0)

I recommend implementing as per our other tools.